.yarnrc.yml

```diff
@@ -1,11 +1,17 @@
 nodeLinker: node-modules
 
+enableGlobalCache: false
+
+nmMode: hardlinks-local
```

Comment:
This one is not totally required. We'll the ultimate choice made by yarn 4. But I'd say... keep it there explicitly (in yarn 3 it's the default).